smrf/distribute/precipitation.py

```diff
@@ -268,6 +278,7 @@ def distribute(self, data, dpt, precip_temp, ta, time, wind, temp,
 
         self._logger.debug('%s Distributing all precip' % data.name)
         data = data[self.stations]
+        data = self.modifier.scale_input_data(data)
```

Comment:
It is being run twice if `marks2017` is used, but on two separate datasets so it will not be compounding. 

I'm not sure how to implement this in `data` without getting into a specific implementation for each of the `InputData.load_class` options. If I understand the procedure correctly, the hrrr data timesteps are read in at runtime for each distribution timestep. https://github.com/USDA-ARS-NWRC/smrf/blob/master/smrf/framework/model_framework.py#L409

Running this operation on each timestep is not ideal, but the alternative was having unique logic for station and hrrr data, so I leaned towards the former. I can switch it if you want though, I'm not stuck on this approach. 
One option is scaling it here https://github.com/USDA-ARS-NWRC/smrf/blob/master/smrf/framework/model_framework.py#L330
and then a loop to scale input data by variable after this line NWRC/smrf/blob/master/smrf/framework/model_framework.py#L409

Or I could pass a `Modifier` object into the `InputData` class and implement a `load` method on the `InputData` class that enforces that the input data is modified. This would mean moving the `modifier` property to image data now and it just wouldn't do anything for the other variables. 

What is your preference?

Reply:
I do like that as a place for the input scaling. That means the modifiers will need to be initialized within the `InputData` class as well, but that doesn't seem like a bad thing.

Reply:
The code I just pushed accomplishes this.